CHANGELOG.md

```diff
@@ -1,50 +1,128 @@
-## Parse Server Changelog
-<!--
-    Please make sure you add your feature at the bottom of the associated group.
-    Groups should be ordered:
-    - NEW (EXPERIMENTAL)
-    - NEW
-    - IMPROVE
-    - FIX
-    Thanks for contributing to Parse Server!
--->
-
-### master
-[Full Changelog](https://github.com/parse-community/parse-server/compare/4.5.0...master)
+# Parse Server Changelog
+## Current Major Version
```

Comment:
My only nit is this heading - it seems a bit verbose to me & visually a bit much.

Reply:
Which one

Reply:
Yes I like that 👌